app/controllers/api/v1/users_controller.rb

```diff
@@ -4,7 +4,7 @@ module Api
   module V1
     class UsersController < ApiController
       skip_before_action :verify_authenticity_token # TODO: amir - Revisit this.
-
+      before_action :default_language, only: :create
```

Comment:
This shouldn't be done in the before action - it should be done directly in the create action